concent-builder/test.yml

```diff
@@ -0,0 +1,37 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: "{{ shared_user }}"
+      block:
+        - name: Configure kubectl to operate on the right cluster
+          command: gcloud container clusters get-credentials \
+              "{{ gke.cluster }}"                            \
+              --project "{{ gke.project }}"                  \
+              --zone    "{{ gke.zone }}"
+
+        - name:    Create custom network that allows communication between django and postgres containers
+          command: docker network create {{ cluster }}-{{ image_prefix }}unittest-network
+
+        - name:    Run postgres container for django unit tests
+          command: docker run --rm                                                                               \
+              --detach                                                                                           \
+              --net={{ cluster }}-{{ image_prefix }}unittest-network                                             \
```

Comment:
`--net` works just like other options. No need for `=`. And you could use the full form: `--network`

Reply:
Done.